src/main/java/openblocks/Config.java

```diff
@@ -380,6 +380,10 @@
 	@ConfigProperty(category = "devnull", name = "sneakClickToGui", comment = "If true, /dev/null will require sneaking in addition to clicking air to open gui")
 	public static boolean devNullSneakGui = true;
 
+	@OnLineModifiable
+	@ConfigProperty(category = "hangglider", name = "enableThermal", comment = "Enable a whole new level of hanggliding experience through thermal lift. See keybindings for acoustic vario controls")
+	public static boolean hanggliderEnableThermal = true;
```

Comment:
I see what you mean. I was hoping for 'true', since there are some awesome modpacks in 1.7.10 that are still missing for later versions (especially gregtech-based ones), but I can also change that in the settings for my worlds.
You should definitely try it out before deciding, though. It's a lot of fun and you need some skill to travel further distances since the effective heigth is limited to ~85 < y < ~127.